### PR TITLE
Fix captions

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

**Review by marinaserranomontes — APPROVED** [review]
lgtm

**Review by IGitGotIt — APPROVED** [review]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 2.27.4  (April 2024)
+
+- [Update]: Update OpenTok Android SDK and OpenTok iOS SDK to version 2.27.1.
+
+  This version fixes some crashes.
+
+  This version the `OTSubscriber captionReceived` event handler.
```
Comment (jintgeorge) [inline CHANGELOG.md:7]: captionReceived or captionsReceived. Also the line seems to be incomplete.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 2.27.4  (April 2024)
+
+- [Update]: Update OpenTok Android SDK and OpenTok iOS SDK to version 2.27.1.
```
Comment (jintgeorge) [inline CHANGELOG.md:3]: The previous version says Vonage Video iOS SDK and this one says Opentok SDK. Is that intentional?. Is this version update done in another PR, not seeing this version change in this PR.

**Review by jintgeorge — CHANGES_REQUESTED** [review]
Left some comments.

File: CHANGELOG.md (hunk 1 above)
Comment (jeffswartz) [reply CHANGELOG.md]: It is `captionReceived`, like in the web client SDK. I fixed the incomplete sentence.

**Review by jintgeorge — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # 2.27.5  (May 2024)
```
Comment (jeffswartz) [inline CHANGELOG.md:1]:
```suggestion
# 2.27.5  (June 2024)
```
